.eslintrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
 
         // Coding Style
         'indent'                   : [2, 4, {'SwitchCase': 1, 'MemberExpression': 0}],
-        'linebreak-style'          : [2, 'unix'],
+        'linebreak-style'          : [2, 'windows'],
```

Comment:
I'm not sure that windows is a good idea :thinking: 

If I develop under Linux (I only use it) my code will not pass the lint ...